1-development/list_posts_by_rating/app/controllers/api/v1/posts_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Api::V1::PostsController < ApplicationController
+  def index
```

Comment:
Why create multiple controllers and not use methods from the same controller?

Reply:
I like the way DHH uses Controllers. Based on Bootcamp. It makes it more uniform, organized, isolated, more REST and I can only have the basic functions (CRUD) in each of the controllers.